deploy/kubernetes/shared-settings.yaml

```diff
@@ -41,6 +41,8 @@ UPLOADED_PEDIGREE_FILE_RECIPIENTS: ''
 # this allows cluster administration only from known machines or from machines on google cloud.
 MASTER_AUTHORIZED_NETWORKS: '69.173.112.0/21,69.173.127.232/29,69.173.127.128/26,69.173.127.0/25,69.173.127.240/28,69.173.127.224/30,69.173.127.230/31,69.173.120.0/22,69.173.127.228/32,69.173.126.0/24,69.173.96.0/20,69.173.64.0/19,69.173.127.192/27,69.173.124.0/23'
 
+TERRA_API_CACHE_EXPIRE_SECONDS: 3600
```

Comment:
I don't think this is an acceptable amount of time to cache things. If someone had their access revoked in Terra it seems so unacceptable to continue to access seqr for an hour. Unless you have received explicit sign off from the compliance team about using such a long cache, please make this 60